src/schedule/zephyr_ll.c

```diff
@@ -57,6 +57,8 @@ static void zephyr_ll_assert_core(const struct zephyr_ll *sch)
 static void zephyr_ll_task_done(struct zephyr_ll *sch,
```

Comment:
The patch prefix is a bit odd as this is not touching any chain-dma code.

Reply:
@kv2019i oops, yeah, a left-over from the original version... Too bad this cannot be changed in a "low impact" way - without re-running the CI. But I guess yes, it's wrong enough to deserve a re-push

Reply:
@lyakh let's change. we have not been perfect with the prefixes for sure, but I think we set a better example here